.identity/.terraform/providers/registry.terraform.io/hashicorp/azuread/2.30.0/darwin_amd64

```diff
@@ -0,0 +1 @@
+/Users/pasqualedevita/.terraform.d/plugin-cache/registry.terraform.io/hashicorp/azuread/2.30.0/darwin_amd64
```

Comment:
I think this shouldn't be here. same for `.identity/.terraform/providers/registry.terraform.io/hashicorp/azurerm/3.33.0/darwin_amd64`

Reply:
good catch! done in 1f6dcd0826e1691c6d14405cb1883f1bd7c72c29